dcm4che-tool/dcm4che-tool-stowrs/src/main/java/org/dcm4che3/tool/stowrs/StowRS.java

```diff
@@ -200,7 +200,8 @@ private void stowMetaDataAndBulkData(File file) {
             try {
                 metadata = parseJSON(file.getPath());
             } catch (Exception e) {
-                LOG.error("error parsing metadata JSON file {}", file);
+                LOG.error("error parsing metadata JSON file {}: {}", file, e.getMessage());
+                e.printStackTrace();
```

Comment:
Hi, Kevin!
As we are already using a Logger here, I wouldn't use e.printStackTrace() (which will print to standard out).
I believe if you do it like this the stack trace should also be visible:
`LOG.error("error parsing metadata JSON file {}: {}", file, e);`
Can you try that out please.